packages/create-plugin/src/utils/utils.handlebars.ts

```diff
@@ -12,8 +12,12 @@ export const ifEq = (a: any, b: any, options: HelperOptions) => {
 
 export const normalizeId = (pluginName: string, orgName: string, type: PLUGIN_TYPES) => {
   const re = new RegExp(`-?${type}$`, 'i');
-  const newPluginName = pluginName.replace(re, '');
-  return changeCase.paramCase(orgName) + '-' + changeCase.paramCase(newPluginName) + `-${type}`;
+  const pluginNameRegex = new RegExp('[^0-9a-z]', 'g');
```

Comment:
These two regex are the same you can reuse one. what about `nameRegex``

Reply:
Right, https://github.com/grafana/plugin-tools/pull/95/commits/8cb1fc172605d6dedc615a4c0dfc9a22c5899cb8